odpi/egeria
# Helm8

Latest updates to helm charts

Installing the chart will now
 - deploy & configure kafka/zookeeper (3 node)
 - deploy egeria, configure egeria into cohort
 - deploy atlas, configure atlas into cohort
 - provide a debug node for watching kafka messages

ie these pieces are all hooked together and working - for example one can create a new entity in egeria and see it in atlas :-)

In addition other components are deployed, but not yet configured.
